HLTrigger/Configuration/python/CustomConfigs.py

```diff
@@ -200,3 +200,8 @@ def L1XML(process,xmlFile=None):
     process.ESPreferL1TXML = cms.ESPrefer("L1TUtmTriggerMenuESProducer","L1TriggerMenu")
 
     return process
+
+def CTPPSRun2Geometry(process):
+    if hasattr(process,'ctppsGeometryESModule')
```

Comment:
```suggestion
    if hasattr(process,'ctppsGeometryESModule'):
```

Reply:
It is also necessary to return `process` as well as actually use the customization. Using the commit at https://github.com/cms-sw/cmssw/commit/1364f2692566ecdea01d4852e8ebd880ccceedad on top of this PR allows the following to complete without errors in a local test:

- `runTheMatrix.py -l 11725.0,11925.0,limited --ibeos`
- `addOnTests -j 8`

Reply:
thansk @forthommel @christopheralanwest .
@Martin-Grunewald is `Configuration/HLT/python/addOnTestsHLT.py` automatically generated from `HLTrigger/Configuration/test/cmsDriver.csh` ? Do we need to change both manually?

Reply:
They are separate, both need the change.

Reply:
ok, I've added 1364f26